project/src/main/career-data.gd

```diff
@@ -21,6 +21,21 @@ const MAX_DISTANCE_TRAVELLED := 999999
 ## The maximum number of consecutive levels the player can play in one career session.
 const HOURS_PER_CAREER_DAY := 8
 
+## Array of dictionaries containing milestone metadata, including the necessary rank, the distance the player will
+## travel, and the UI color.
+const RANK_MILESTONES := [
+	{"rank": 64.0, "distance": 1, "color": Color("48b968")},
+	{"rank": 36.0, "distance": 2, "color": Color("48b968")},
+	{"rank": 24.0, "distance": 3, "color": Color("48b968")},
+	{"rank": 20.0, "distance": 4, "color": Color("78b948")},
+	{"rank": 16.0, "distance": 5, "color": Color("b9b948")},
+	{"rank": 10.0, "distance": 10, "color": Color("b95c48")},
+	{"rank": 4.0, "distance": 15, "color": Color("b94878")},
+	{"rank": 0.0, "distance": 25, "color": Color("b948b9")},
+]
+
+const RANK_MILESTONE_FAIL := {"rank": 64.0, "distance": 0, "color": Color("bababa")}
```

Comment:
Document this constant

Reply:
Added comment; ` ## The rank milestone to display when the player fails a boss level.`